mmv1/products/beyondcorp/api.yaml

```diff
@@ -0,0 +1,252 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Beyondcorp
+display_name: Google BeyondCorp
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://beyondcorp.googleapis.com/v1/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Beyondcorp API
+    url: https://console.cloud.google.com/apis/library/beyondcorp.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: 'AppConnector'
+    description: |
+      A BeyondCorp AppConnector resource represents an application facing component deployed proximal to 
+      and with direct access to the application instances. It is used to establish connectivity between the 
+      remote enterprise environment and GCP. It initiates connections to the applications and can proxy the 
+      data from users over the connection.
+    min_version: ga
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/beyondcorp-enterprise/docs/enable-app-connector'
+      api: 'https://cloud.google.com/beyondcorp/docs/reference/rest#rest-resource:-v1.projects.locations.appconnectors'
+    base_url: projects/{{project}}/locations/{{region}}/appConnectors
+    self_link: projects/{{project}}/locations/{{region}}/appConnectors/{{name}}
+    create_url: projects/{{project}}/locations/{{region}}/appConnectors?app_connector_id={{name}}
+    delete_url: projects/{{project}}/locations/{{region}}/appConnectors/{{name}}
+    update_url: projects/{{project}}/locations/{{region}}/appConnectors/{{name}}
+    update_verb: :PATCH
+    update_mask: true
+    # iam_policy: !ruby/object:Api::Resource::IamPolicy
+    #   parent_resource_attribute: name
+    #   method_name_separator: ':'
+    #   fetch_iam_policy_verb: :GET
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        input: true
```

Comment:
If you mark this url_param_only this won't be sent in the request body or read from the response body, removing the need for the custom expand/flatteners. Usually this is what we want as the `name` field is almost always ignored in request bodies, and can't change so it doesn't matter if we read it back